openapi/external_api.yaml

```
@@ -194,10 +194,29 @@ paths:
         "400":
           $ref: "#/components/responses/BadRequest"
         "404":
-          $ref: "#/components/responses/ClusterNotFound"
+          $ref: "#/components/responses/NotFoundResponse"
         "500":
           $ref: "#/components/responses/InternalError"
 
+  /clusters/{runtimeID}/config/{version}:
+    get:
+      description: "Get cluster configuration"
+      parameters:
+        - name: runtimeID
+          required: true
+          in: path
+          schema:
+            type: string
+            format: uuid
+        - name: version
+          required: true
+          in: path
+          schema:
+            type: string
+      responses:
+        "200":
+          $ref: "#/components/responses/configurationOkResponse"
```

One small suggestion:

```suggestion
          $ref: "#/components/responses/ConfigurationOkResponse"
```

---

I will apply it in https://github.com/kyma-incubator/reconciler/pull/437